platform_config.toml

```
@@ -36,3 +36,7 @@ app_id = { release = "org.mozilla.focus", beta = "org.mozilla.focus.beta", night
 [platform.monitor_cirrus]
 is_glean_app = true
 app_id = { release = "monitor.cirrus" }
+
+[platform.vpn]
+is_glean_app = true
+app_id = { desktop = "mozillavpn", ios = "org_mozilla_ios_firefoxvpn", android = "org_mozilla_firefox_vpn"}
```

```suggestion
[platform.mozilla_vpn]
is_glean_app = true
app_id = { release = "mozillavpn" }
```

I believe the official app name is `mozilla_vpn`.
`app_id` is used to specify the datasets if an app has multiple channels. VPN only has release.

---

We might need separate entries for each of the VPN apps

---

For context: the main place where we need this mapping is for monitoring rollouts that are defined in Experimenter. The rollouts specify the `appId` name in their targeting, so OpMon has to map it back to a specific application and channel

---

can I combine multiple platforms in one opmon dashboard?

---

Yes. Either by creating a custom data source that unions them, or by having separate metrics for each platform and combining them via metric groups

---

so if I create datasources ... I only have the platform proforma and because the tool otherwise complains? 
sounds great thank you.